README.md

```diff
@@ -8,6 +8,15 @@
 
 We used Python 3.6.4 to make our stuff.
 
+## Install dependencies
+
+- From the command prompt, enter:
+    ```bash
+    python3 -m venv .venv
+    source .venv/bin/activate
+    pip install -r requirements.txt
+    ```
+
```

Comment:
what. is. that.

Reply:
these are three commands you have to run to get termcolor installed into the project